templates/views/moduleDefinition.jade

```diff
@@ -40,21 +40,40 @@ block topLevelPanels
                                 table.table.table-striped.entry-detail-table
                                     tr
                                         td 
-                                            | Instance
+                                            | Instance : 
                                             a(href='http://sbolstandard.org/v2#Module',title='Learn more about Modules') 
                                                 span.fa.fa-info-circle
-                                        td 
-                                            | Definition
+                                            | &nbsp;
+                                            | Definition 
                                             a(href='http://sbolstandard.org/v2#definition',title='Learn more about definitions') 
                                                 span.fa.fa-info-circle
+                                            | &nbsp;
+                                            | (
+                                            | MapsTos 
+                                            a(href='http://sbolstandard.org/v2#MapsTo',title='Learn more about MapsTos') 
+                                                span.fa.fa-info-circle
+                                            | &nbsp;
+                                            | )
                                     tr
                                         td
                                             for module in meta.modules
                                                 a(href=module.url) #{module.name}
-                                                br
-                                        td
-                                            for module in meta.modules
+                                                | &nbsp;
+                                                | :
+                                                | &nbsp;
                                                 a(href=module.definition.url) #{module.definition.name}
+                                                if module.mapsTos.length > 0
+                                                  | (
+                                                  - var first = true
+                                                  for mapsTo in module.mapsTos
```

Comment:
This should be in JavaScript, not jade.

Reply:
I know what you mean, and it was tricky to figure out how to do it.  However, moving to Javascript would mean precomputing the display string.  Not bad I guess.  I could be moved to summarize perhaps.  I think I'm going to merge this for now though.